addons/mail/static/src/utils/test_utils.js

```diff
@@ -393,8 +396,21 @@ async function createRootComponent(self, Component, { props = {}, target }) {
  *   the View class to use in createView.
  * @param {Object} [param0.viewOptions] makes only sense when `param0.hasView`
  *   is set: the view options to use in createView.
- * @param {boolean} [param0.waitUntilMessagingInitialized=true]
+ * @param {string} [param0.waitUntilMessaging='initialized'] Determines the
```

Comment:
This variable reads well as a sentence when it's assigned a value, but its name is otherwise a bit unclear.
I would consider naming it `waitUntilMessagingCondition` or `waitUntilMessagingState`?

Reply:
chosen `waitUntilMessagingCondition`